translations/base-ind.yaml

```diff
@@ -4,17 +4,17 @@ steamPage:
         yang meluas tanpa batas.
     discordLinkShort: Official Discord
     intro: >-
-        Shapez.io is a relaxed game in which you have to build factories for the
-        automated production of geometric shapes.
+        Shapez.io adalah game santai dimana anda harus membuat pabrik untuk 
```

Comment:
Still has trailing space, everything else is fine.

Reply:
Note: I have no knowledge of these languages, i'm just critiquing the yaml syntax and stuff.